spynl/tests/test_log_error.py

```diff
@@ -0,0 +1,138 @@
+"""Test functions from spynl.main."""
+import pytest
+
+from spynl.main.utils import log_error
+
+
+@pytest.fixture
+def logger(monkeypatch):
+    """Patch a number of things so we can return a logger and not have any
```

Comment:
I really need to install that pydocstyle thing because I gotta break a habit for this :)